ql/src/java/org/apache/hadoop/hive/ql/parse/TransformSpec.java

```diff
@@ -22,7 +22,7 @@
 public class TransformSpec {
 
   public enum TransformType {
-    IDENTITY, YEAR, MONTH, DAY, HOUR, TRUNCATE, BUCKET, VOID
+    IDENTITY, YEAR, MONTH, DAY, HOUR, TRUNCATE, BUCKET, VOID, UNKNOWN
```

Comment:
do we really need unknown? shouldn't we fail if some spec transform is unsupported?

Reply:
Removed.